app/src/main/res/layout/security_popup.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: line break before the license header. Android Studio messes this up when using the auto-formatter. :)